app/views/pages/help.html.slim

```diff
@@ -1,60 +1,23 @@
-markdown:
-  # Help
-
-  ## Program Guides
-  
-  [Student Application Guide](http://railsgirlssummerofcode.org/students/application/)
-
-  [Coaches Guide](http://railsgirlssummerofcode.org/guide/coaching/)
-
-  [Mentor's Guide](http://railsgirlssummerofcode.org/guide/projects/)
-
-  ## Site and account help
-
-  ### Quick guides
-
-  [How to create a team](create-a-team)  
-
-  [How to fill out your application](fill-out-your-application)  
-
-  ### Your own account
-
-  Signing in through GitHub will create a user account on our side. If your account has already been added by one of your team mates (for example, if they've added you to a team) then you'll claim your account
-  by signing in through GitHub.
-
-  Once signed in, you can update your profile information. Please tell the
-  community a little bit about you. Why are you participating, what you
-  hope to get out of this, etc.
-
-  ### Your team's profile
-
-  On the [teams list](/teams) check if your team has already been added. If you're
-  already part of a team then you can edit your team's profile.
-
-  If you haven't been added to your team, yet, then ask one of the members to
-  sign in and add you. They will need your GitHub account name for that.
-
-  With your team's profile you and your team can tell the community about your project,
-  how and where you plan to work, ... everything that might be useful or fun to know :)
-
-  If your team has a GitHub organization (you might want one and they're free
-  for Open Source) then please add the handle. If your team has a Twitter
-  account, then please add it, too. *Note: you can only add these once your team has officially been accepted as a RGSoC team.*
-
-  ### Your team's sources
-
-  Your team can register "sources" from which this application will try to
-  aggregate updates; please add any blog that is relevant for your team by registering its RSS or Atom feed URLs. 
-
-  Please also add all the GitHub repository URLs that you are planning to
-  work on as sources.  Currently this won't actually do anything yet, but we
-  plan to aggregate information from there, too.
-
-  ### Just ask!
-
-  If you think something is unclear or should be improved, or if you find a
-  bug, then please [report an issue](https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues).
-
-  Or email us at [contact@rgsoc.org](mailto:contact@rgsoc.org)
-
-  Or [tweet](https://twitter.com/RailsGirlsSoC) us :)
+h1 Help
+
+.col-md-4
+  h2 Site and account help
+  ul
+   li = link_to "How to create an account", "create-an-account"
+   li = link_to "How to create a team", "create-a-team"
+   li = link_to "How to confirm your coach status", "confirm-a-coach"
+   li = link_to "How to fill out your application", "fill-out-your-application"
+
+.col-md-4
+  h2 Program guides
+  ul
+   li = link_to "Student's Application Guide", "http://railsgirlssummerofcode.org/students/application/"
```

Comment:
could you remove the `'s` here? so just “Student Application Guide”, “Coach Guide”, “Mentor Guide”.